clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/FilesetCatalogCache.java

```diff
@@ -80,6 +107,14 @@ private Cache<NameIdentifier, FilesetCatalog> newCatalogCache(
         .build();
   }
 
+  private Cache<NameIdentifier, Fileset> newFilesetCache(
+      ScheduledThreadPoolExecutor filesetCleanScheduler) {
+    return Caffeine.newBuilder()
+        .maximumSize(100)
+        .scheduler(Scheduler.forScheduledExecutorService(filesetCleanScheduler))
```

Comment:
What is the usage of this `scheduler`? I don't find any `Callable` or `Runnable` for this scheduler.

Reply:
ok, if it's unnecessary, I'll remove all the scheduler from the client side.

BTW, it's first introduced by the PR #4320 in [GravitinoVirtualFileSystem.java](https://github.com/apache/gravitino/pull/4320/files#diff-64c5b87c772c66c642144ad44764c1339ad846ddc013000c52c9ab503cbce02eR141-R145)